3rdparty/openenclave/ert.patch

```diff
@@ -665,22 +725,38 @@ index cf814d36a..5de0c930d 100644
 +        oe_sgx_enclave_properties_t* const props_payload =
 +        (oe_sgx_enclave_properties_t*)(oeimage_payload.elf.image_base + oeimage_payload.elf.oeinfo_rva);
 +        memset(props_payload->sigstruct, 0, sizeof(props_payload->sigstruct));
++
++        // get payload data offset and size from EDG-extended oeinfo section
++        const uint64_t* const p =
++            (uint64_t*)(oeimage_payload.elf.image_base + oeimage_payload.elf.oeinfo_rva + 0x800);
++        payload_data_offset = p[0];
++        oeimage_payload.elf.payload_data_size = p[1];
```

Comment:
Not really experienced with common habits in C++ for pointer arithmetic, but this looks weird to me. Maybe use suggestion above and define these offsets somewhere?